src/index.js

```diff
@@ -0,0 +1,56 @@
+import './index.css';
+
+// const taskSection = document.querySelector('section');
+const taskInput = document.querySelector('input');
+const taskContainer = document.querySelector('.task-container');
+// const clearBtn = document.querySelector('button');
```

Comment:
- [ ] **Required:** For a professional-looking code base, kindly remove unused and commented out code. This improves code readability and follows the clean code convention.